packages/extension/public/locales/fr/translation.json

```diff
@@ -96,7 +96,7 @@
   "Before we start, just a couple of notes regarding use:": "Avant de commencer, quelques remarques concernant l'utilisation:",
   "Aleph Zero Signer does not send any clicks, pageviews, and events to external servers": "Nous n'envoyons aucuns clics, pages vues ou événements à un serveur central",
   "Aleph Zero Signer does not use any analytics": "Nous n'utilisons aucun outil de suivi ou d'analyse",
-  "All your data stays on this device–your secret phrases, addresses, and any other information are only stored locally": "Nous ne collectons aucune clé, adresse ou aucune information - vos informations ne quittent jamais cette machine",
+  "All your data stays on this device:\nyour secret phrases, addresses, and\nany other information are only stored locally": "Nous ne collectons aucune clé, adresse ou aucune information - vos informations ne quittent jamais cette machine",
```

Comment:
I'm wondering if we want to maintain the new lines in the translations. Guess it's worth it, but then for the more exotic to us languages this can get quite difficult 🤔

Reply:
What about [those](https://github.com/Cardinal-Cryptography/aleph-zero-wallet-extension/pull/20/files/8b2ccd6c172957474d0d0f7693b7aad0495e6e08..72b80d9188f35a4a789cf921549a48fbce18c5a2#diff-e8a81fe45ffb8abb978bae354165ad1229882df83c69caa83ffbf22dd82fe156R98)? Is that a remaining of some sort after an upstream or sth like that?

Reply:
I think it would be the best to clear them for now. What do you think?

Reply:
Confirmed with Jakub that it can be cleared :D